test/test_functional.py

```diff
@@ -0,0 +1,133 @@
+import os
+
+import torch
+import torchaudio
+import unittest
+import test.common_utils
+
+
+class TestFunctional(unittest.TestCase):
+    data_sizes = (2, 20)
+    number_of_trials = 10
+
+    def _compare_estimate(self, sound, estimate):
+        # trim sound for case when constructed signal is shorter than original
+        sound = sound[..., :estimate.size(-1)]
+
+        self.assertTrue(sound.shape == estimate.shape, (sound.shape, estimate.shape))
+        self.assertTrue(torch.allclose(sound, estimate, atol=1e-4))
+
+    def _test_istft_helper(self, sound, kwargs):
```

Comment:
The goal of the test would be clearer if the name was `_test_istft_is_inverse_of_stft`. Similarly for tests invoking this.